lib/modules/datasource/metadata.spec.ts

```diff
@@ -231,6 +232,50 @@ describe('modules/datasource/metadata', () => {
     ]);
   });
 
+  describe('massageUrl', () => {
+    it('Should return an empty string when massaging an invalid url', () => {
+      expect(massageGithubUrl('not a url')).toMatch('');
+    });
+
+    test.each`
+      sourceUrl
+      ${'git@github.com:user/repo'}
+      ${'http://github.com/user/repo'}
+      ${'http+git://github.com/user/repo'}
+      ${'https+git://github.com/user/repo'}
+      ${'ssh://git@github.com/user/repo'}
+      ${'git://github.com/user/repo'}
+      ${'https://www.github.com/user/repo'}
+      ${'https://user.github.com/repo'}
```

Comment:
is this a valid url?

Reply:
Doesn't seem to be

Reply:
I added that test case for the GitHub pages branch although those URLs are not valid anymore:

https://github.blog/changelog/2021-01-29-github-pages-will-stop-redirecting-pages-sites-from-github-com-after-april-15-2021/

Yet, I’m not sure if it should be fixed in this issue or create another issue because it is going out of scope

Reply:
ok, leave it for now.